container-compose.yml

```diff
@@ -101,12 +112,16 @@ services:
       DATABASE_URL: postgresql://forem:forem@db:5432/PracticalDeveloper_development
     volumes:
       - .:/opt/apps/forem:z
-    entrypoint: ["dockerize", "-wait", "tcp://db:5432", "-wait", "tcp://redis:6379", "-wait", "http://rails:3000", "-timeout", "300s"]
     command: ["bundle", "exec", "sidekiq","-c","2"]
 
   db:
     image: postgres:11-alpine
     container_name: forem_postgresql
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -d forem -U forem" ]
```

Comment:
Does this need `CMD-SHELL`? Can it use `CMD` like `redis-cli` does below?

Reply:
Sure we can. It should be better even as a container not always has a SHELL (/bin/shell)  prompt. I'll fix it.